src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int NUMBER_OF_BALLS = 3;
+
+    public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            System.out.println("ball №" + (i + 1) + lottery.getRandomBall());
```

Comment:
Your message should be completely in toString() method. No need to add anything here
```suggestion
            System.out.println(lottery.getRandomBall());
```

Reply:
For me so do better. I receive more information. Where is problem?